src/site/content/en/learn-core-web-vitals/index.njk

```diff
@@ -1,6 +1,56 @@
 ---
-layout: path
+layout: collection
 override:tags: []
 date: 2020-09-16
 pathName: learn-core-web-vitals
+cards:
+  - title: Thinking on ways to solve cafe wall illusion
+    description: Praesent accumsan eros orci quis congue metus porta a sed dapibus magna nunc non finibus erat ullamcorper sit amet.
+    thumbnail: image/SZHNhsfjU9RbCestTGZU6N7JEWs1/MqeKXDtUPO7zszzpRXcF.png
+    eyebrow:
+      icon: featured
+      text: Featured
+    cssClass: 'card-vertical'
+  - title: 'The CSS Podcast: Media query range syntax'
+    description: Praesent accumsan eros orci quis congue metus porta a sed dapibus magna.
+    thumbnail: image/SZHNhsfjU9RbCestTGZU6N7JEWs1/VwL892KEz6bakZMlq10D.png
+    eyebrow:
+      icon: podcast
+      text: Podcast
+    cssClass: 'card-horizontal col-2 bg-tertiary'
+  - title: Meet the Chrome team
```

Comment:
This type of card needs to link to /meet-the-team page, or generally have ability to add a target url to a card.

Reply:
I have added this feature.